frontend/src/pages/Testing.tsx

```diff
@@ -115,6 +118,24 @@ const Dashboard = () => {
             Get {process.env.REACT_APP_PAYMENT_TOKEN_NAME || ''} Tokens
           </button>
         </div>
+
+        <div>
+          <button
+            onClick={async () => {
+              console.log('adding tokens', process.env);
+              walletContext.walletAddToken(
+                process.env.REACT_APP_PAYMENT_TOKEN_CONTRACT_ADDRESS as string,
```

Comment:
Instead of re-typing, can you actually do the coercion?

`String(process.env.REACT_APP_PAYMENT_TOKEN_CONTRACT_ADDRESS)`

This will give us safety around the type instead of just forcing TS to accept it as a string.

Reply:
Same with below

Reply:
ohh I never learned about the difference but yeah lemme do that